reframe/core/modules.py

```diff
@@ -451,11 +453,11 @@ def is_module_loaded(self, module):
 
     def load_module(self, module):
         self._exec_module_command('load', str(module),
-                                  msg='could not load module %s' % module)
+            msg="could not load module '%s' correctly" % module)
```

Comment:
Identation is not correct here: `msg` should align at the column where the first function argument is. Otherwise, you should start the function arguments in a new line.

Reply:
Done.